ckanext/zarr/assets/css/zarr.css

```diff
@@ -133,8 +133,12 @@
     color: #198A00;
 }
 
-.dataset-search-form .btn-default:hover > .fa-search {
-    color: #198A00;
+#dataset-search-form .btn-default:hover > .fa-search {
+    color: #198A00 !important;
```

Comment:
Generally using `!important` is not ideal in CSS. Why do we need it here?

Reply:
Yes, it is because the .fa-search has a different color in other search inputs:
```
.fa-search {
    color: #198A00;
}

#dataset-search-form .btn-default:hover > .fa-search {
    color: #198A00 !important;
}

#dataset-search-form .btn-default > .fa-search {
    color: #DFF0DB !important;
}
```

Reply:
@ChasNelson1990

Reply:
But the more specific one should take priority _without_ using `!important`?